### DISCUSSION TITLE
Is this plugin supposed to be supported by 10.9.6 servers?

### DISCUSSION THREAD

**Discussion by LazerDickMcCheese** [lyarenei/jellyfin-plugin-listenbrainz#97] [Q&A] [answered]
Just installed this plugin, but it is listed as not supported. I read on a bug report that this was due to a Jellyfin problem a month or so ago, is that still the case? And is there an ETA for 10.9.6+ support?

**Comment by lyarenei** [comment]
> Just installed this plugin, but it is listed as not supported.

The major plugin version (ex. 4.a.b.c) should always be compatible with all patch versions of a specific minor version of Jellyfin (ex. 10.9.x). 

That's of course unless Jellyfin team decided to make some breaking changes in the patch version (they usually do so on minor versions - like 10.8.x -> 10.9.x). I am not aware of any compatibility issues as of now - or more exactly I did not receive any reports and I am still on Jellyfin 10.8.x.

> I read on a bug report that this was due to a Jellyfin problem a month or so ago, is that still the case?

These were only related to initial support and have been fortunately resolved before JF 10.9.0 was released.

> And is there an ETA for 10.9.6+ support?

As I mentioned earlier and as far as I know, the plugin should be compatible. If you have the `4.0.0.2` plugin version installed and it does not work with JF 10.9.6, please create a bug report as that needs to be fixed.

Additionally, you can find a compatible version of the plugin for your server in the [readme](https://github.com/lyarenei/jellyfin-plugin-listenbrainz?tab=readme-ov-file#install-via-repository).

**Reply by LazerDickMcCheese** [reply, marked as answer]
First, let me apologize for this. It turns out my server's repository listed the current version as 2.3.0...somehow I must have messed that up. I removed and re-added the repo, now it's current and working as intended.
